closed/OpenJ9.gmk

```diff
@@ -438,7 +438,6 @@ ifeq (true,$(OPENJ9_ENABLE_CMAKE))
 
   CMAKE_ARGS += -DJ9VM_OPT_METHOD_HANDLE=OFF
   CMAKE_ARGS += -DJ9VM_OPT_OPENJDK_METHODHANDLE=ON
-  CMAKE_ARGS += -DJ9VM_OPT_OPENJDK_THREAD_SUPPORT=ON
```

Comment:
Are you going to remove the J9VM_OPT_OPENJDK_THREAD_SUPPORT cmake option from OpenJ9 as well after the extensions changes are merged?

Reply:
My bad, I see https://github.com/eclipse-openj9/openj9/pull/16521 now.